front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -152,6 +152,45 @@
         word-break: break-all;
       }
 
+      .not-opened {
+        position: relative;
+        overflow: hidden;
+        max-height: 3.75rem;
+        max-width: 100%;
+        span {
+          padding-inline: 0.2188rem;
+          position: absolute;
+          bottom: 0;
+          right: 0;
+          background: rgba(222, 221, 214, 1);
+          color: var(--grey80);
+        }
+      }
+
+      .opened {
+        position: absolute;
+        top: 0;
+        background-color: var(--white100);
+        color: var(--grey80);
+        padding: 3.375rem 3.5rem 3.125rem 3.5rem;
+        font-size: 1rem;
+        font-weight: 400;
+        line-height: 1.5rem;
+        z-index: 3;
+        border-radius: 0.375rem;
+        box-shadow:
+          0rem 0.375rem 1.3125rem -0.3125rem rgba(255, 171, 88, 0.31),
+          0rem 1rem 1.875rem -0.3125rem rgba(0, 0, 0, 0.19),
+          0rem 0.1875rem 0.3125rem -0.125rem rgba(0, 0, 0, 0.16);
+        transform: translateX(-0.5rem);
```

Comment:
Instead of a `transform`, could we use a `left` position maybe?

Reply:
yes, it's done